test/acceptance/accessibility.py

```diff
@@ -160,7 +160,7 @@ def test_staff_grade(self):
 
         # Refresh the page, then verify accessibility of the Staff Grade section (marked Complete).
         self.browser.refresh()
-        self._verify_staff_grade_section(self.STAFF_GRADE_EXISTS, None)
+        self._verify_staff_grade_section(self.STAFF_GRADE_EXISTS)
```

Comment:
This method was updated to only take 1 parameter; I forgot to run a11y tests locally before merging so this method call snuck through.